doc/user-guide/Bootstrap-Scripts.md

```diff
@@ -18,3 +18,57 @@ Environment variables, available from scripts:
 
 - `ERTS_PATH` - path to Erlang Runtime System, used by MongooseIM.
 - `MIM_DIR` - MongooseIM release installation directory.
+
+
+# Templating bootstrap script
+
+The script `bootstrap20-template.escript` renders files from the `templates/` directory and writes
+result files into the `etc/` directory. If you need the result files in a separate directory,
+create another script `bootstrap30-template.sh`, that moves files into a proper location.
+
+
+The `etc/templates.ini` file contains default template variables.
+
+
+A template config example:
+
+```erlang
+[options]
+demo_session_lifetime = 600
+demo_tls_versions = 'tlsv1.2', 'tlsv1.3'
+```
+
+Only lowercase variables allowed in `templates.ini`.
+
+You can redeclare options using environment variables when executing the bootstrap script:
+
+```bash
+MIM_DEMO_SESSION_LIFETIME=700 mongooseimctl bootstrap
+```
+
+Environment variables should have a `MIM_` prefix. The variable names can be uppercase.
```

Comment:
> The verialbe names can be uppercase.

Does it mean that if some put `MIM_demo_SESSION_lifetime=700` it will also work?

Reply:
Yes... Though I would suggest not to do it xdd